rules/anti-static/elf/entropy.yara

```diff
@@ -1,22 +1,14 @@
-import "math"
```

Comment:
If I were to paint this shed, I would always place `import` statements before `include`.

YARA allows either order, but I prefer following Python's style and it makes some sense for `include` to be next to the other rules.

Not a block, just a nit :)

Appreciate that variable names are being converted to snake case too.

Reply:
No disagreements here. `include` after `import` is technically more correct since it's adding the included file's contents to the file which would follow `import` if it were added directly to the file to begin with.

Updated in: [`cb0a939` (#937)](https://github.com/chainguard-dev/malcontent/pull/937/commits/cb0a939dd04b1a3be0d630f053b5b9afdcb7b6a6).